ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/ReadService.scala

```diff
@@ -63,75 +64,65 @@ trait ReadService extends ReportsHealth {
     *   meta-data.
     *   Note that the ledger time of unrelated updates is not necessarily monotonically
     *   increasing.
+    *   The creating transaction need not have a [[Update.TransactionAccepted]] even on this participant
+    *   if the participant does not host a stakeholder of the contract, e.g., in the case of divulgence.
     *
     * - *time skew*: given a [[Update.TransactionAccepted]] with an associated
     *   ledger time `lt_tx` and a record time `rt_tx`, it holds that
     *   `rt_TX - minSkew <= lt_TX <= rt_TX + maxSkew`, where `minSkew` and `maxSkew`
-    *   are parameters specified in the ledger [[TimeModel]].
```

Comment:
Note for reviewers: a good chunk of the changes on the guarantees of `ReadService.stateUpdates` is cleanup that is in principle independent of this PR. It's probably easiest to view the file as a whole and check that commentary makes sense to you.